writing/example.tex

```diff
@@ -0,0 +1,203 @@
+\documentclass{article}
+\usepackage{color}
+\usepackage{listings}
+
+
+% see listings documentation
+\lstdefinelanguage{slim}{
+    % Eidos language keywords from 
+    % https://github.com/MesserLab/SLiM/blob/4bcc36a02aeacdc9ee808e38d62836f854246502/eidos/eidos_token.h#L90
+    morekeywords=[1]{if,else,do,while,for,in,next,break,return,function},
+    % SLiM callback keywords from
+    % https://github.com/MesserLab/SLiM/blob/4bcc36a02aeacdc9ee808e38d62836f854246502/core/slim_eidos_block.cpp#L32
+    morekeywords=[2]{first,early,late,initialize,mutationEffect,fitnessEffect,interaction,mateChoice,modifyChild,recombination,mutation,survival,reproduction},
+    % Other special SLiM tokens from
+    % https://github.com/MesserLab/SLiM/blob/4bcc36a02aeacdc9ee808e38d62836f854246502/QtSLiM/QtSLiMSyntaxHighlighting.cpp#L294
+    morekeywords=[3]{sim,community,slimgui,
+        p0,p1,p2,p3,p4,p5,p6,p7,p8,p9,p10,p11,p12,p13,p14,p15,p16,p17,p18,p19,p20,p21,p22,p23,p24,p25,
+        p26,p27,p28,p29,p30,p31,p32,p33,p34,p35,p36,p37,p38,p39,p40,p41,p42,p43,p44,p45,p46,p47,p48,p49,p50,
+        g0,g1,g2,g3,g4,g5,g6,g7,g8,g9,g10,g11,g12,g13,g14,g15,g16,g17,g18,g19,g20,g21,g22,g23,g24,g25,
+        g26,g27,g28,g29,g30,g31,g32,g33,g34,g35,g36,g37,g38,g39,g40,g41,g42,g43,g44,g45,g46,g47,g48,g49,g50,
+        m0,m1,m2,m3,m4,m5,m6,m7,m8,m9,m10,m11,m12,m13,m14,m15,m16,m17,m18,m19,m20,m21,m22,m23,m24,m25,
+        m26,m27,m28,m29,m30,m31,m32,m33,m34,m35,m36,m37,m38,m39,m40,m41,m42,m43,m44,m45,m46,m47,m48,m49,m50,
+        s0,s1,s2,s3,s4,s5,s6,s7,s8,s9,s10,s11,s12,s13,s14,s15,s16,s17,s18,s19,s20,s21,s22,s23,s24,s25,
+        s26,s27,s28,s29,s30,s31,s32,s33,s34,s35,s36,s37,s38,s39,s40,s41,s42,s43,s44,s45,s46,s47,s48,s49,s50,
+        i0,i1,i2,i3,i4,i5,i6,i7,i8,i9,i10,i11,i12,i13,i14,i15,i16,i17,i18,i19,i20,i21,i22,i23,i24,i25,
+        i26,i27,i28,i29,i30,i31,i32,i33,i34,i35,i36,i37,i38,i39,i40,i41,i42,i43,i44,i45,i46,i47,i48,i49,i50},
+    sensitive=true,
+    morecomment=[l]{//},
+    morestring=[b]",
+}
+% colors from 
+% https://github.com/MesserLab/SLiM/blob/4bcc36a02aeacdc9ee808e38d62836f854246502/QtSLiM/QtSLiMSyntaxHighlighting.cpp#L139
+% numberLiteralFormat.setForeground(inDarkMode ? QColor(115, 145, 255) : QColor(28, 0, 207));
+% stringLiteralFormat.setForeground(inDarkMode ? QColor(220, 98, 90) : QColor(196, 26, 22));
+% commentFormat.setForeground(inDarkMode ? QColor(90, 210, 90) : QColor(0, 116, 0));
+% identifierFormat.setForeground(inDarkMode ? QColor(70, 205, 216) : QColor(63, 110, 116));
+% keywordFormat.setForeground(inDarkMode ? QColor(220, 83, 185) : QColor(170, 13, 145));
+\definecolor{slimstring}{RGB}{196,26,22}
+\definecolor{slimcomment}{RGB}{0,116,0}
+\definecolor{slimidentifier}{RGB}{63,110,116}
+\definecolor{slimkeyword}{RGB}{170,13,145}
+\definecolor{slimstage}{RGB}{40,40,40}
```

Comment:
Has the PDF been regenerated to show this?  I'd prefer for it to look the way it does in SLiMgui, I think; I tried syntax-coloring these and I didn't like the result so I disabled it.  And anyhow there's an argument for consistency.

Reply:
I also think using a shade of gray de-emphasizes the event/callback type, which I'm not sure is a good idea since that is key information.  If you really wanted to syntax color these, a color that pops _more_ than black, rather than less, would seem like a better choice to me.  But like I said, I experimented with all this back in the day and (personally, obvs this is a matter of taste) didn't like the results.

Reply:
okay done